samples/correlation-csharp/FunctionAppCorrelation/EntityOrchestration.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Newtonsoft.Json.Linq;
+
+namespace FunctionAppCorrelation
+{
+    /// <summary>
+    /// This example is testing for Entity works on the new correlation implementation.
```

Comment:
```suggestion
    /// This example is for testing that Durable Entities work with the new correlation implementation.
```

Reply:
Changed! Sorry, I would have committed this suggestion directly from the UI tool, but I didn't catch that option until now :)